tests/parsing/test_semantic_model_parsing.py

```diff
@@ -334,6 +420,37 @@ def test_semantic_model_primary_time_dimension_parsing() -> None:
     assert dimension.type_params is not None
 
 
+def test_base_semantic_model_dimension_parsing() -> None:
+    """Test parsing base attributes of PydanticDimension object."""
+    description = "Test sematic_model dimension description"
```

Comment:
This is the cutest copy/paste gif I could find.....

![cat on copier](https://media.giphy.com/media/oCCLHVNt8YO64/giphy.gif)

Reply:
Not marking comment as resolved because cat gif 🐈